src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -105,7 +105,7 @@ public void doPost_urlNotInDatabase_shouldAddToDatabaseAndReturnRedirect() throw
     servlet.doPost(request, response);
 
     assertThat(datastoreService.prepare(new Query(LectureUtil.KIND)).countEntities()).isEqualTo(1);
-    verify(response).sendRedirect("/view/?id=1&video-id=wXhTHyIgQ_U");
+    verify(response).sendRedirect("/view/?id=1");
```

Comment:
here too, extra slash
```suggestion
    verify(response).sendRedirect("/view?id=1");
```